topology/probes/istio/destinationrule.go

```diff
@@ -48,7 +51,21 @@ func newDestinationRuleProbe(client interface{}, g *graph.Graph) k8s.Subprobe {
 }
 
 type destinationRuleSpec struct {
-	App string `mapstructure:"host"`
+	App           string `mapstructure:"host"`
```

Comment:
Because `host` is the name of the relevant service that implements an application with the same name. See here: `https://istio.io/docs/reference/config/networking/v1alpha3/destination-rule/`

Reply:
But still I would suggest that if the yaml file uses `host` then that is mapped to a field name `Host`

Reply:
`destinationRuleSpec` is not available anymore, which makes this comment irrelevant.